src/Yoti.Auth/DigitalIdentity/GetQrResult.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json;
+
+
+namespace Yoti.Auth.DigitalIdentity
+{
+    public class GetSessionResult
```

Comment:
this file name is `GetQrResult.cs` but the class name is `GetSessionResult`. Can u explain?

Reply:
File names were mixed and corrected now